spec/models/preservation_copy_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe PreservationCopy, type: :model do
+  let!(:preservation_object2) do
+    PreservedObject.create!(druid: 'ab123cd45679', version: 1, preservation_policy: 'keepit', size: 1)
+  end
+  let!(:endpoint2) { Endpoint.create!(endpoint_name: 'oracle', endpoint_type: 'cloud') }
+  let!(:preservation_copy) do
+    PreservationCopy.create!(
+      preserved_object_id: preservation_object2.id, endpoint_id: endpoint2.id, version: 0, status: 'fixity_error'
+    )
+  end
+
+  it 'is not valid without valid attributes' do
+    expect(PreservationCopy.new).not_to be_valid
+  end
```

Comment:
Same as per my comment above - is there a sane and non-anal way to test that the correct attributes are present without generating all possible permutations?